src/openforms/logging/tests/test_events.py

```diff
@@ -1,12 +1,23 @@
 from django.test import TestCase
 from django.utils.translation import gettext as _
 
+from openforms.forms.models.form import FormLogic
 from openforms.logging import logevent
 from openforms.logging.models import TimelineLogProxy
 from openforms.submissions.tests.factories import SubmissionFactory
+from openforms.submissions.tests.mixins import VariablesTestMixin
 
 
-class EventTests(TestCase):
+class EventTests(VariablesTestMixin, TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        cls.submission = SubmissionFactory.from_data(
+            {
+                "firstname": "foo",
+                "birthdate": "2022-06-20",
+            },
+        )
```

Comment:
Try to keep your test setup/data _in_ the test itself rather than splitting it over multiple test case methods. I'd really just put this inside of `def test_submission_logic_evaluated`